dev-docs/modules/goldfishAdsRtdProvider.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: page_v2
+title: GoldfishAds Real Time Data Provider
+display_name: GoldfishAds Real Time Data Provider
+description: GoldfishAds Real Time Data Provider 
+page_type: module
+module_type: rtd
+module_code: goldfishAdsRtdProvider
+enable_download: true
+vendor_specific: true
+sidebarType: 1
+---
+
+# GoldfishAds Real Time Data Provider
+
+## Integration
+
+1. Compile the Adnuntius RTD Module and Adnuntius Bid Adapter into your Prebid build:
```

Comment:
Suspect the mention of adnuntius here is a copy-paste error

Reply:
Also, it may be helpful to include some contact info so interested publishers can contact you.